elections.md

```diff
@@ -101,4 +91,16 @@ treated as “brand free” activities.
 ## Emeritus Term
 
 Members of the steering committee will graduate to becoming Emeritus members of
-the steering committee.
\ No newline at end of file
+the steering committee.
+
+## Steering committee and election officer recusal
+
+Steering committee members and the appointed election offers pledge to recuse
+themselves from any form of electioneering, including campaigning, nominating,
+endorsing, or even asking people to run.
+
+[kubernetes/community/events/elections/README.md]: https://git.k8s.io/community/events/elections/README.md
+[voting exception form]: https://www.surveymonkey.com/r/k8s-sc-election-2018
```

Comment:
I think all these links need to be added to the body of the doc too.  Or maybe this does that.  I don't know MD very well.